Utils.lua

```diff
@@ -27,7 +27,7 @@ local function capitalize(string)
 end
 
 local classOrdered = {
-    "Death Knight", "Druid", "Hunter", "Mage", "Priest", "Rogue", "Shaman", "Paladin", "Warlock", "Warrior"
+    "Deathknight", "Druid", "Hunter", "Mage", "Priest", "Rogue", "Shaman", "Paladin", "Warlock", "Warrior"
```

Comment:
I do not like this change. I understand the whole issue happens because how CLM treates classes, but instead of making it this way I would rather rework the whole class treatment everywhere so the issue won't come up at all.
This means no longer base on the class canonical name to display but display actual names and use canonicals to filter/search.